### PR TITLE
Add option to align title text in window

### PR DESCRIPTION
**Pull request by Rhaoma**
I felt that i really needed a basic way to adjust the alignment of the title text, but since it is drawn directly it wasn't really possible. I therefore came up with this basic way of letting the user choose between the LV_TXT_FLAG's and changing the drawing bounds depending on said flag. Mind you that I'm very new to all of this so it might not be the most optimal way of doing it.
I have of course written the alignments as I felt was most suitable, but I shouldn't be the judge of that.
Current alignment boundaries: 

LV_TXT_FLAG_CENTER makes the text align in the center of the header but ensures it can’t overextend into to header button area;

LV_TXT_FLAG_RIGHT makes the text align at the right side, but takes the right side header buttons into account

LV_TXT_FLAG_FIT & LV_TXT_FLAG_EXPAND I wasn’t too sure about what to do so as of now it just aligns them as normal

LV_TXT_FLAG_NONE Is equal to no flag set by the user and therefore I have just set it to the default coords, like normal. The text then align at the left side.

Hope it works, and it's useful enough

### CODE REVIEW

File: src/lv_widgets/lv_win.h [hunk 1]
```diff
@@ -57,6 +57,7 @@ typedef struct {
     lv_obj_t * header;                /*Pointer to the header container of the window*/
     char * title_txt;                 /*Pointer to the title label of the window*/
     lv_coord_t btn_w;                 /*Width of the control buttons*/
+    uint8_t title_txt_align;          //Control the alignment of the header text
```
Comment (kisvegabor) [inline src/lv_widgets/lv_win.h]: Please use `/* */` type comments

**Review by kisvegabor — COMMENTED** [review]
Looks good in general, thank you! 

The only thing is missing 2 API functions to set/get alignment. E.g. `lv_win_set/get_title_align()`.

Could you add them?

File: src/lv_widgets/lv_win.h (hunk 1 above)
Comment (Rhaoma) [reply src/lv_widgets/lv_win.h]: Sure, but why do you prefer this type of comment: `/* */`

Comment (embeddedt) [reply src/lv_widgets/lv_win.h]: We used to support C89 which required this type of comment. Due to consistency, it's stuck around as part of the project's [coding style](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md#comments).

**Comment by Rhaoma** [comment]
Hope its what you were after. By the way i modeled the names after `lv_win_btn_get_alignment()` but it may not be exactly the name you want

**Comment by kisvegabor** [comment]
Looks good, thank you!

**Pull request merged**